opencensus-shim/build.gradle

```diff
@@ -0,0 +1,6 @@
+description = 'OpenConsensus OpenCensus Bridge'
+
+dependencies {
+    compile project(':openconsensus-api')
```

Comment:
I thought the idea was to keep the name of the package. So it's implementation will simply replace the API

Reply:
Correct, that's what I thought as well.

Reply:
> I thought the idea was to keep the name of the package. So it's implementation will simply replace the API

That's right, but before that we need to make sure the openconsensus implementations are compatible with opencensus APIs. This "opencensus-shim" is a POC that we can re-direct all opencensus APIs to openconsensus APIs before we move on to implementations.

Reply:
I would do it differently:
1. Copy the current OpenCensus API.
1. For example for the abstract class Tagger, we will change it to a final class that just redirects to OpenConsensus (probably some composition is required).

Reply:
I see, I'll post a new PR to make that change. (Though this is very similar to the idea in this PR, for example [TaggerShim](https://github.com/bogdandrutu/openconsensus/pull/111/files#diff-8ea1957831a8721b2db1e827e5d64bf1R24) is a final class that implements all the abstract methods in Tagger.)